### PR TITLE
Add Mumbai's recommendations

### PR DESCRIPTION
**Pull request by pritish-devurkar**
This Pull Requests closes issue #53.
### Adding restaurants and acitivies
- Does anyone have any favorite cuisines?
- Indoor or outdoor?

### CODE REVIEW

File: findpritz-MumbaiCity.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Mumbai City :heart: :heart: :heart:
+
+#### Restaurants
```
Comment (n0rbs) [inline findpritz-MumbaiCity.md:3]: @findpritz What's Mumbai's most popular restaurant? 🐔 From Singapore 😄

Comment (pritish-devurkar) [inline findpritz-MumbaiCity.md:3]: @n0rbs The Taj Hotel! :smile:

Comment (RameshKatkam) [inline findpritz-MumbaiCity.md:3]: @n0rbs  VadaPav is my favorite. :)

Comment (pritish-devurkar) [inline findpritz-MumbaiCity.md:3]: :+1:  :+1:  :+1:

File: findpritz-MumbaiCity.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+####Things to do
```
Comment (WilliamRClark) [inline findpritz-MumbaiCity.md:7]: /play bueller

**Comment by pritish-devurkar** [comment]
:+1: :shipit:

**Pull request merged**